### PR TITLE
Add checks

### PR DESCRIPTION
**Pull request by daPhipz**
See #39.

### CODE REVIEW

**Comment by daPhipz** [comment]
Weird... It doesn't do any checks for me...

**Comment by daPhipz** [comment]
Is the `script_functionality` test working correctly now? I wonder why it shows only one error message when we check whether the `gettext` package is installed or not...

I will fix the shellcheck errors next.

**Comment by daPhipz** [comment]
OK, so this is as far as I get at the moment. Funnily enough, when I run `shellcheck` against `cloc.sh` locally on my machine, it doesn't throw any error at all...
I have the fear of breaking things so please check for yourself what needs to be done here.
We could of course bump up the severity level of `shellcheck` (from highest to lowest: `error`, `warning`, `info` and `style`) as well and "fix" it that way.

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -57,4 +57,32 @@ jobs:
             exit 1
           else
             awk -v a=$CURRENT -v b=$BASE 'BEGIN {printf "Percentage increased! (%3.4f%)\n", (a-b)}'
-          fi
\ No newline at end of file
+          fi
+
+  shellcheck:
```
Comment (Schneegans) [inline .github/workflows/checks.yml:62]:
```suggestion
  shellcheck:
    name: Run ShellCheck
```

File: .github/workflows/checks.yml [hunk 2]
```diff
@@ -57,4 +57,32 @@ jobs:
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run shellcheck
```
Comment (Schneegans) [inline .github/workflows/checks.yml:66]:
```suggestion
      - name: Run ShellCheck
```

File: .github/workflows/checks.yml [hunk 3]
```diff
@@ -57,4 +57,32 @@ jobs:
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check error conditions # Currently, gettext is not installed. Both if statements should be false.
+      run: |
+        if $GITHUB_WORKSPACE/compile-locales.sh; then
+          if $GITHUB_WORKSPACE/update-locales.sh; then
+            exit 1
+          fi
+        fi
+    - name: Check correct workflow # Now, gettext will be installed and both IVERTED if statements should be false.
```
Comment (Schneegans) [inline .github/workflows/checks.yml:81]:
```suggestion
    - name: Check correct workflow # Now, gettext will be installed and both INVERTED if statements should be false.
```

File: .github/workflows/checks.yml [hunk 4]
```diff
@@ -57,4 +57,32 @@ jobs:
...
+
+  shellcheck:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run shellcheck
+        run: |
+          find $GITHUB_WORKSPACE -type f -and \( -name "*.sh" \) | xargs shellcheck
+
+  script_functionality:
```
Comment (Schneegans) [inline .github/workflows/checks.yml:70]:
```suggestion
  script_functionality:
    name: Check Script Functionality
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -26,6 +26,7 @@ Emoji | Description
 :rocket: `:rocket:` | When you improved performance.
 :memo: `:memo:` | When you wrote documentation.
 :beetle: `:beetle:` | When you fixed a bug.
+:heavy_check_mark: `:heavy_check_mark:` | When you worked on checks or adjusted the code to be compliant with them
```
Comment (Schneegans) [inline CONTRIBUTING.md:29]:
```suggestion
:heavy_check_mark: `:heavy_check_mark:` | When you worked on checks or adjusted the code to be compliant with them.
```

**Review by Schneegans — CHANGES_REQUESTED** [review]
Looks good! Just some minor remarks from my side. And I tested the changes ShellCheck suggests for `cloc.sh` and they seem to work.

**Review by Schneegans — APPROVED** [review]

**Pull request merged**